drivers/src/pci/bus.rs

```diff
@@ -81,8 +82,8 @@ impl PciBus {
 
             // Avoid double mutable borrow of device_arena by re-acquiring the reference to the bridge
             // device here. PciBus::enumerate() may have added devices and re-allocated the arena.
-            match device_arena.get_mut(bridge_id) {
-                Some(PciDevice::Bridge(bridge)) => {
+            match *device_arena.get(bridge_id).unwrap().lock() {
```

Comment:
Unwrap might need comment unless obvious from context.

Reply:
This one should be clear from the comment immediately above it (+ the context of having just looked up the device a few lines up).